docs/reference/ml/apis/put-dfanalytics.asciidoc

```diff
@@ -43,7 +43,18 @@ and {stack-ov}/built-in-roles.html[Built-in roles].
   
 `dest` (Required)::
   (object) The destination configuration, consisting of `index` and optionally 
-  `results_field` (`ml` by default).
+  `results_field` (`ml` by default). The destination index will be automatically 
+  created if it does not exist. The `index.number_of_shards` and 
+  `index.number_of_replicas` settings of the source index will be copied over the 
+  destination index. When the source index matches multiple indices, these 
+  settings will be set to the maximum values found in the source indices.
+  
+  The mappings of the source indicies are also attempted to being copied over
+  `dest`, however, if the mappings of any of the fields don't match among the 
+  source indicies, the attempt will fail with an error message.
+  
+  If `dest` already exists, then it will be use as is. This makes it possible to 
```

Comment:
rephrase:  `If 'dest' already exists` -> `If the destination index already exists`, as `dest` refers to the entire object which in the future might not need an `index` to be specified.